zoo-prometheus/rules/high-cpu.yml

```
@@ -0,0 +1,11 @@
+groups:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)